Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.6
+
+COPY . /usr/src/app/
+WORKDIR /usr/src/app
+
+RUN pip install --no-cache-dir --requirement requirements/production.txt
+
+ENV DJANGO_SETTINGS_MODULE="meetpy.settings.docker"
+
+EXPOSE 8000
+
+WORKDIR meetpy
+CMD python ./manage.py runserver
```

Comment:
I do not think this command will work. The proper use of CMD is a list of args:
```["python", "manage.py", "runserver"]```

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#cmd

Also you need django to be running dev server on IP 0.0.0.0

So: ```CMD ["python", "manage.py", "runserver", "0.0.0.0:8000"]```

Reply:
Thanks, it works now